docs/getting_started/hopsworksai/guides/getting_started_azure.rst

```diff
@@ -0,0 +1,289 @@
+.. _getting-started-hopsworks-ai:
+
+Getting started with Hopsworks.ai (Azure)
+=========================================
+
+Hopsworks.ai is our managed platform for running Hopsworks and the Feature Store
+in the cloud. It integrates seamlessly with third party platforms such as Databricks,
+SageMaker and KubeFlow. This guide shows how to set up Hopsworks.ai with your organization's Azure account.
+
+.. contents:: :local:
+
+Step 1: Connecting your Azure account
+-----------------------------------
+
+Hopsworks.ai deploys Hopsworks clusters to your Azure account. To enable this, you have to
+create a service principal and a custom role for Hopsworks.ai granting access
+to either a subscription or resource group.
+
+Step 1.1: Creating a service principal for Hopsworks.ai
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+On Hopsworks.ai, go to *Settings/Cloud Accounts* and choose to *Configure* Azure:
+
+.. _connect-azure-0.png: ../../../_images/connect-azure-0.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-0.png
+    :alt: Cloud account settings
+    :target: `connect-azure-0.png`_
+    :align: center
+    :figclass: align-center
+
+Select *Add subscription key*:
+
+.. _connect-azure-0.1.png: ../../../_images/connect-azure-0.1.png
+.. figure:: ../../../imgs/hopsworksai/connect-azure-0.1.png
+    :alt: Add subscription keys
+    :target: `connect-azure-0.1.png`_
+    :align: center
+    :figclass: align-center
+
+The Azure account configuration will show you the required steps and permissions.
+Ensure that you have the Azure CLI installed (`Install the Azure CLI <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli>`_)
+and are logged in (`Sign in with Azure CLI <https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli>`_).
+
+Copy the Azure CLI command to add a service principal for Hopsworks.ai from the first step and open a terminal:
```

Comment:
This sentence hurt my head :p
I think I would simplify it by just saying:
`Copy the Azure CLI command from the first step and open a terminal:`

Reply:
done